src/components/OrgPostCard/components/PostEditModal.tsx

```
@@ -0,0 +1,269 @@
+                  */}
+                  <video
+                    controls
+                    data-testid="video-preview"
+                    aria-label="Video preview - captions not available for uploaded videos"
+                  >
+                    <source
+                      src={sanitizeDataUrl(attachment.url, 'video/')}
+                      type={attachment.mimeType}
+                    />
+                    {t('videoNotSupported')}
+                  </video>
```

Biome (2.1.2)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)